app/src/main/res/drawable/ic_status_denied.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="56dp"
+    android:height="56dp"
+    android:viewportWidth="56"
+    android:viewportHeight="56">
+  <path
+      android:pathData="M23.5053,25H32.4947L45.9489,25.6511C46.5386,25.6796 47,26.1043 47,26.6186V29.3814C47,29.8957 46.5386,30.3204 45.9489,30.3489L32.4947,31L23.5053,31L10.0511,30.3489C9.4614,30.3204 9,29.8957 9,29.3814L9,26.6186C9,26.1043 9.4614,25.6796 10.0511,25.6511L23.5053,25Z"
+      android:fillColor="#FA5B5B"/>
```

Comment:
Why don't you use a universal color for two themes?

Reply:
Don't forget about the license

Reply:
in some places there are single resource for both themes